### ISSUE TITLE
More details and features for the Reactor map?

### ISSUE THREAD

**Issue by WorldSpinAround** [studio-minus/ppg-bugs-and-requests#3043]
### Feature description

I am aware things are left to be completed, as shown in the big screen indicating the reactor's characteristics.
But since it's a Preview, I thought some feedback would be welcome.

-I feel the Reactor could get some more complexity, notably inside. Currently, it's hollow, and except cooking ragdolls, there isn't much going on.
-Apparently there is a cooling rod system? No idea how that works, but it would be sweet.
-A way to reattach the big tubes? Would be neat in an emergency situation.
-A shutdown for the reactor?
-More parts for the reactor itself. Currently, it's two huge blocks of texture, and... I feel for its size, more could be pulled away from it.
-Obviously the reactor need more consequences, the electricity system and also the meltdown/breakdown of it. Could pump temperature to 100k+, nothing happens.
-Is there a fuel to it? Maybe a system to control its input.
-I don't know what the two tubes are for, but with their removal, temperature shuts down. I'm going to assume they are the cooling rods. (I thought it was the ones dipping into the hot fluid under). They need some work, for many textures go through them, the tube itself can go through walls... But yeah, my main suggestion for it is to blow, like, stream of cooling. Anyone in its path get frozen solid. Assuming that is how the tubes works in the, huh, 'LORE'.
-Ah right. The Reactor parts aren't impacted by Black holes. Maybe this could change?
-I hope in the case of an emergency, the player can try to fix the issue by any means necessary. I know there are cooling guns and devices, perhaps multiple of them could make the difference? If it isn't, perhaps making the reactor susceptible to them could be fun.
-And who knows, maybe you have more plans, more depth and mechanisms than what we saw currently. I'm all out for original concept and fun system!

That's about it. :)

In short, Reactors can be complex, but there is a big potential to make this one in the game really fun to play around with.

### Feature purpose

Again, the map is in Preview Version, and I am assuming you seek Feedback for it, even if some might be getting work. Still, I thought it was only fair. :)
